src/app/core/services/data-integration/videos-store.service.ts

```diff
@@ -0,0 +1,99 @@
+import { Injectable } from '@angular/core';
+import { BehaviorSubject } from 'rxjs';
+import _ from 'lodash';
+
+import { SavedVideoData } from '../../../shared/interfaces/saved-video-data.interface';
+import { LOCAL_STORAGE_VIDEOS_KEY } from '../../../shared/constans/local-storage-keys';
+import { StorageService } from '../utils/storage.service';
+import { SnackbarService } from '../utils/snackbar.service';
+import { DEFAULT_VIDEOS_DATA } from '../../../shared/constans/demo-videos';
+import {
+  ALL_VIDEOS_REMOVED_MESSAGE,
+  DEMO_VIDEOS_ADDED_MESSAGE,
+  DEMO_VIDEOS_ALREADY_IN_LIBRARY,
+  VIDEO_ADDED_MESSAGE, VIDEO_ADDED_TO_FAVOURITES_MESSAGE,
```

Comment:
Już nie chciało się enterów wciskać? :trollface: Zaimportuj to jako `as something` jak jest za dużo tych importów.

Reply:
A to sam webstorm sobie zrobił :trollface: